src/lib/crud.class.ts

```diff
@@ -185,7 +185,10 @@ export class CrudBase extends Vue implements List {
         this.deleteDialog = false;
     }
 
-    protected getSelectedKeys(uniqueKey: string = 'key'): string[] {
+    protected getSelectedKeys(uniqueKey: string = 'key'): GenericObject[] | string[] {
+        if(this.selected[0].original){
```

Comment:
Ezt jobb lenne Type alapjan, mert ez egy base method es mindenhol hasznalva lesz / van.
Tehat ha typeof this.selected[0] === TreeNodeType